src/analyzer/analyzerebur128.cpp

```diff
@@ -76,5 +76,5 @@ void AnalyzerEbur128::storeResults(TrackPointer tio) {
     mixxx::ReplayGain replayGain(tio->getReplayGain());
     replayGain.setRatio(db2ratio(fReplayGain2));
     tio->setReplayGain(replayGain);
-    qDebug() << "ReplayGain 2.0 (libebur128) result is" << fReplayGain2 << "dB for" << tio->getLocation();
+    qDebug() << "ReplayGain 2.0 (libebur128) result is" << fReplayGain2 << "dB for" << tio->getFileInfo();
```

Comment:
The old one was readable without knowing the insights the new one not. 
tio->getFileInfo().location(); may work but I think we should keep the wrapper in this case.

Reply:
Why? In this context you shouldn't need to know or care which one of the multiple string representations a `TrackFile` can be converted to should be used for printing an informational message.

Reply:
It reads: "hey, since when we can qDebug() a QFileInfo()" 
We should avoid to confuse our contributors like that.

Reply:
QFileInfo() becomes "qDebuggable" in Qt 5.12 (or 5.10 or 5.11?) so why should this be confusing? The implementation already uses this feature for Qt >= 5.12.

Reply:
Ok, in that case I can get used to it.
It is anyway not too important, it was intialy just a stumbling block for me.